src/sidebar/components/HypothesisApp.js

```diff
@@ -172,6 +172,7 @@ function HypothesisApp({ auth, bridge, settings, session, toastMessenger }) {
     >
       {route !== 'notebook' && (
         <TopBar
+          // @ts-expect-error - This type mismatch needs to be fixed or the prop removed.
```

Comment:
Fixing this error is not trivial, so I've punted on it here. I think the best fix will be to remove the `auth` prop from `TopBar` and `HelpPanel` entirely, since both can get the information they need from the store.